java/feathub-udf/flink-udf/src/main/java/com/alibaba/feathub/flink/udf/aggregation/AggregationFunction.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 The Feathub Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.feathub.flink.udf.aggregation;
+
+import org.apache.flink.table.types.DataType;
+
+import java.io.Serializable;
+
+/** Interface of aggregation function. */
+public interface AggregationFunction<IN_T, OUT_T> extends Serializable {
```

Comment:
Can we make the names/locations of the existing classes (e.g. TimeWindowedFirstValueAggFunc) consistent with classes newly added in this PR?